app/sentinel/busywait/initialization/run.go

```diff
@@ -53,8 +55,38 @@ import (
 func RunInitCommands(ctx context.Context) {
 	cid := ctx.Value("correlationId").(*string)
 
+	src, acquired := spiffe.AcquireSourceForSentinel(ctx)
+
+	if !acquired {
+		timeout := env.InitCommandRunnerWaitTimeoutForSentinel()
+
+		timeoutCtx, cancel := context.WithTimeout(ctx, timeout)
+		defer cancel()
+
+		ticker := time.NewTicker(10 * time.Second)
+		defer ticker.Stop()
+
+		for {
+			select {
+			case <-timeoutCtx.Done():
+				log.ErrorLn(cid, "Failed to acquire source at RunInitCommands (1)")
```

Comment:
I thought of using `Fatalln()` here and kill the app; but even if it cannot run init commands, it might still function, so I made a sacrifice of stability over reliability. 

This case should ideally never happen anyways.